SWTableViewCell/PodFiles/SWTableViewCell.m

```diff
@@ -91,12 +89,7 @@ - (void)initializer
     // Add the cell scroll view to the cell
     UIView *contentViewParent = self;
     UIView *clipViewParent = self.cellScrollView;
-    if (![NSStringFromClass([[self.subviews objectAtIndex:0] class]) isEqualToString:kTableViewCellContentView])
```

Comment:
AT here, index out of range because of `[self.subviews objectAtIndex:0]` call towards `self.subviews` which is void.

Reply:
If you remove this part of the code, the cell is not clickable, swipe doesn't work, but you can build app will run, no crash.